### ISSUE TITLE
WFLYMSG0075 and libaio

### ISSUE THREAD

**Issue by sixcorners** [jboss-dockerfiles/wildfly#32]
I get this warning when starting wildfly in domain mode:

    [Server:server-one] 17:28:14,240 WARN  [org.jboss.as.messaging] (MSC service thread 1-5) WFLYMSG0075: AIO wasn't located on this platform, it will fall back to using pure Java NIO. Your platform is Linux, install LibAIO to enable the AIO journal.

I guess it's a HornetQ thing. Would including libaio in the image be a good idea? It seems that if you use it with the wrong filesystem it can slow things down.

**Comment by spolti** [comment]
Yes, it is the hornetQ journal, the base CentOS image does not have the libaio installed, all you need to do is:
Start the wildfly docker image, then:

docker exec -it --user root <CONTAINER_ID_OR_NAME> /bin/sh -c 'yum install libaio -y'
Example:

docker run --name wildfly -it jboss/wildfly  /opt/jboss/wildfly/bin/domain.sh -b 0.0.0.0 -bmanagement 0.0.0.0

docker exec -it --user root wildfly /bin/sh -c 'yum install libaio -y'

Restart the container then it will be automatically enabled:

[org.hornetq.core.server] (ServerService Thread Pool -- 62) HQ221012: Using AIO Journal
